api/app/clients/GoogleClient.js

```
@@ -563,71 +546,38 @@ class GoogleClient extends BaseClient {
 
-  async getCompletion(_payload, options = {}) {
-    const { parameters, instances } = _payload;
-    const { onProgress, abortController } = options;
-    const streamRate = this.options.streamRate ?? Constants.DEFAULT_STREAM_RATE;
-    const { messages: _messages, context, examples: _examples } = instances?.[0] ?? {};
-
-    let examples;
-
-    let clientOptions = { ...parameters, maxRetries: 2 };
+  initializeClient() {
+    let clientOptions = { ...this.modelOptions, maxRetries: 2 };
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'clientOptions' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)